docs/agent/changes.md

```diff
@@ -279,6 +275,22 @@ The following options and tags are deprecated:
 
 The `kube_service` tagging depends on the `Datadog Cluster Agent`, which is not released yet.
 
+## Python Modules
+
+While we are continuing to ship the python libraries that shipped with Agent 5,
```

Comment:
"the python libraries that shipped with Agent 5" > "the python libraries that ship with Agent 5" ?